shark/examples/shark_inference/stable_diffusion/resources/model_db.json

```diff
@@ -64,5 +64,197 @@
     "dreamlike/v2_1base/vae/fp32/length_77/untuned":"dl_vae_23dec_fp32",
     "dreamlike/v2_1base/vae/fp32/length_77/untuned/base":"dl_vaebase_23dec_fp32",
     "dreamlike/v2_1base/clip/fp32/length_77/untuned":"dl_clip_23dec_fp32_77"
+  },
+  {
+    "unet": {
+      "tuned": {
+        "fp16": {
+          "cuda": {
+            "compilation_flags": []
+          },
+          "default_device": {
+            "compilation_flags": []
+          }
+        },
+        "fp32": {
+          "cuda": {
+            "compilation_flags": []
+          },
+          "default_device": {
+            "compilation_flags": []
+          }
+        }
+      },
+      "untuned": {
+        "fp16": {
+          "cuda": {
+            "compilation_flags": [
+              "--iree-flow-enable-padding-linalg-ops",
+              "--iree-flow-linalg-ops-padding-size=32",
+              "--iree-flow-enable-conv-nchw-to-nhwc-transform"
+            ]
+          },
+          "default_device": {
+            "compilation_flags": [
+              "--iree-flow-enable-padding-linalg-ops",
+              "--iree-flow-linalg-ops-padding-size=32",
+              "--iree-flow-enable-conv-img2col-transform"
+            ]
+          }
+        },
+        "fp32": {
+          "cuda": {
+            "compilation_flags": [
+              "--iree-flow-enable-conv-nchw-to-nhwc-transform",
+              "--iree-flow-enable-padding-linalg-ops",
+              "--iree-flow-linalg-ops-padding-size=16"
+            ]
+          },
+          "default_device": {
+            "compilation_flags": [
+              "--iree-flow-enable-conv-nchw-to-nhwc-transform",
+              "--iree-flow-enable-padding-linalg-ops",
+              "--iree-flow-linalg-ops-padding-size=16"
+            ]
+          }
+        }
+      }
+    },
+    "vae": {
+      "tuned": {
+        "fp16": {
+          "cuda": {
+            "compilation_flags": [
```

Comment:
My understanding is that tuned models shouldn't need these flow level transform flags because they're already built into the model. This matches with what was already in opt_params but is a good opportunity to remove these. @yzhang93 can you confirm that we don't need these flags?